acs-category-theory-notes.tex

```diff
@@ -14,7 +14,7 @@
 % takes a long time to compile.
 
 \documentclass[a4paper, 11pt, oneside]{book}
-
+    
```

Comment:
Please remove this change. There seem to be a lot of whitespace issues all over the place, trailing and indentation. There should be no trailing whitespace on any line, and all indentation should be done with spaces.